### ISSUE TITLE
`Tree` items can no longer be reordered

### ISSUE THREAD

**Issue by Traveller23** [primefaces/primereact#7156]
### Describe the bug

In previous versions, the items of the `Tree` component could be dragged to change their order, but now they can only be dragged to another item to make them a child of another item.

This issue can be reproduced directly on the official site.

### Reproducer

https://primereact.org/tree/#dragdrop

### System Information

```Shell
System:
    OS: Windows 10 10.0.19045
    CPU: (32) x64 13th Gen Intel(R) Core(TM) i9-13900
    Memory: 54.75 GB / 63.77 GB
  Binaries:
    Node: 21.7.1 - C:\Program Files\nodejs\node.EXE
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 9.4.0 - C:\Program Files\nodejs\pnpm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.19041.4355
  npmPackages:
    primereact: ^10.8.2 => 10.8.2
    react: ^18.3.1 => 18.3.1
    tailwindcss: ^3.4.10 => 3.4.10
```

**Comment by Traveller23** [comment]
> Duplicate of #7058

Okay, when will we be able to use the fixed version?

**Comment by melloware** [comment]
There is no fix ticket #7058 is open so I suggest subscribing to that ticket.

**Comment by Traveller23** [comment]
> There is no fix ticket #7058 is open so I suggest subscribing to that ticket.

Got it, thank you.
